### DISCUSSION TITLE
Percentage display

### DISCUSSION THREAD

**Discussion by seancolsen** [mathesar-foundation/mathesar#1269] [Maintainer discussions]
- When Postgres contains the value `0.5` with the `show_as_percentage` display option set to `true`, how do we display the value to the user?

    - `50%`

        OR

    - `0.5%`

This question came up during today's meeting and @pavish @ghislaineguerin and I would like some clarity from @kgodey @mathemancer @silentninja @dmos62

**Comment by mathemancer** [comment]
I think it should be `50%` in this case, to make math work (i.e., multiplication), and to make it consistent with spreadsheets (at least any spreadsheets I've used).

**Reply by kgodey** [reply]
Agreed.
